tests/e2e/cucumber/environment/index.ts

```diff
@@ -39,6 +40,7 @@ BeforeAll(async function () {
 
 Before(function (this: World, { pickle }: ITestCaseHookParameter) {
   this.feature = pickle
+  console.log('featureName', this.feature.name)
```

Comment:
Dev leftover? Or do you want to log it here?

Reply:
No, I'll take it out. left it out since there were questions

Reply:
Still needs to be removed?

Reply:
removed

Reply:
I can still see it, maybe Florian brought it back :D please remove and check his changes, happy to merge this today/tomorrow if it does what you want 🥳